### PR TITLE
feat: quiz landing page

### PR DESCRIPTION
**Pull request by Shivani-1524**
Added homepage for QuizUp game website.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  object-fit: contain;
+}
+
+.nav-logo {
+  padding: 10px;
+}
+
+.navbar {
+  padding: 5px 30px;
+}
```
Comment (Sahil25061999) [inline index.css]: you can use rem here

File: index.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+.quiz-header {
+  width: 90%;
+  margin: auto;
+  margin-top: 50px;
+  display: grid;
+  grid-template-columns: 60% 40%;
+  min-height: 100px;
+  column-gap: 30px;
```
Comment (Sahil25061999) [inline index.css]: here too.

**Review by Sahil25061999 — APPROVED** [review]
Just add margin for the categories card, rest looks good

**Review by vanshitamoolchandani — APPROVED** [review]
UI looks great, but just change px to rem before merging.

File: index.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  margin: auto;
+  width: 90%;
+  margin-top: 50px;
+}
+
+.quiz-cards {
+  display: grid;
+  gap: 30px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
```
Comment (Ankur9669) [inline index.css:72]: Nice!!!, this min max is awesome

File: index.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fa fa-solid fa-heart"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn show-md">
+                <i class="fas fa-solid fa-bars"></i>
+            </button>
+        </div>
+    </nav>
+    <div class="content-container">
+        <section class="quiz-header">
```
Comment (Ankur9669) [inline index.html:40]: You can use main instead of div

**Review by Ankur9669 — APPROVED** [review]
UI Suggestions:
image

I feel this is not properly aligned

image


You can add some margins here.

Apart from these minor suggestions, the code looks good to merge, Great Work!!

**Review by pradipgharge — APPROVED** [review]
Besides the suggestions given by other peers, everything looks good you can merge it.

**Comment by Shivani-1524** [comment]
> UI Suggestions: image
> 
> I feel this is not properly aligned
> 
> image
> 
> You can add some margins here.
> 
> Apart from these minor suggestions, the code looks good to merge, Great Work!!

Thanks a lot! I will add margins to the cards (no idea why the grid gap isnt working ). Also., I wanted the navbar items to be baselined.

File: index.css (hunk 3 above)
Comment (Shivani-1524) [reply index.css:72]: Thanks 😀

File: index.html (hunk 4 above)
Comment (Shivani-1524) [reply index.html:40]: will change it!

**Comment by Shivani-1524** [comment]
> UI looks great, but just change px to rem before merging.

okay 😀

File: index.css (hunk 1 above)
Comment (Shivani-1524) [reply index.css]: okay will change it

File: index.css (hunk 2 above)
Comment (Shivani-1524) [reply index.css]: okay will change it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+                    <div class="text-card">
+                        <p class="sm-title bold">The Chronicles of Narnia </p>
+                        <p class="xsm-p light-txt">from C.S.Lewis</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="docs-footer white-bg">
+        <p class="sm-p mg-t-15">Made with 💖 by Shivani</p>
```
Comment (gouravmarch20) [inline index.html:99]: no code issue found, may be this is the reason ==>  Made with 💖

**Review by gouravmarch20 — COMMENTED** [review]
no issue

**Comment by amantulsyan35** [comment]
No issues as such, It will be better if there is no margin after the footer, as it is the last section

**Pull request merged**